### ISSUE TITLE
"*" version for libraries is unsafe

### ISSUE THREAD

**Issue by svennam92** [aa7955/clouddatabases-postgresql-helloworld-nodejs#1]
it's best practice to use a wild-card that limits versions to a major/minor version to prevent compatibility issues:

package.json:
```
    "pg": "*",
    "pg-connection-string": "*"
```
to
```
     "pg": "^7.7.1",
     "pg-connection-string": "^2.0.0"
```

**Comment by svennam92** [comment]
https://github.com/aa7955/clouddatabases-postgresql-helloworld-nodejs/pull/2

**Comment by aa7955** [comment]
Merged #2. Thanks @svennam92. I believe that was from the original repo that's out. I'll make sure to change that. Good catch!
